addons/udes_warehouse_classification/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_warehouse_classification,access_warehouse_classification,model_product_warehouse_classification,stock.group_stock_manager,1,1,1,1
```

Comment:
do we need read for stock user on this? think we might

Reply:
Story only mentions manager, can cross the bridge if/when it's on fire, simple change